src/grandorgue/GOFrame.cpp

```diff
@@ -720,7 +720,7 @@ void GOFrame::UpdateTemperamentMenu() {
       }
     }
     wxMenuItem *e = menu->Append(
-      ID_TEMPERAMENT_0 + i, t.GetTitle(), wxEmptyString, wxITEM_CHECK);
+      ID_TEMPERAMENT_0 + i, wxGetTranslation(t.GetTitle()), wxEmptyString, wxITEM_CHECK);
```

Comment:
This is perhaps not the best solution. In older versions of GrandOrgue, `t.GetTitle()` already returned translated temperament names. Please see also https://github.com/GrandOrgue/grandorgue/issues/1104